frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -66,14 +67,14 @@ const GetAccountLink = ({ coinCode, code, name, handleSidebarItemClick }: TGetAc
   const { pathname } = useLocation();
   const active = (pathname === `/account/${code}`) || (pathname.startsWith(`/account/${code}/`));
   return (
-    <div key={code} className="sidebarItem">
+    <div key={code} className={style.sidebarItem}>
       <Link
-        className={active ? 'sidebar-active' : ''}
+        className={active ? style['sidebar-active'] : ''}
         to={`/account/${code}`}
         onClick={handleSidebarItemClick}
         title={name}>
-        <Logo stacked coinCode={coinCode} className="sidebar_icon" alt={name} />
-        <span className="sidebar_label">{name}</span>
+        <Logo stacked coinCode={coinCode} className={style['sidebar_icon']} alt={name} />
+        <span className={style['sidebar_label']}>{name}</span>
```

Comment:
Since you are here, there is a wild mix of camelCase dash and underscore, could you rename all classNames to use camelCase?
so `className={style['sidebar_icon']}` becomes `className={style.sidebarIcon}`

Reply:
I just got why the stacked related styles are global, this CSS affects the Logo component.. 

The only instance where Logo has stacked prop is here in the sidebar. Could you check if it would possible to:
- remove stacked prop
- make stacked CSS part of sidebar.module.css
- use 2 instances of `<Logo>` and pass `style.stacked` here so that the hover effect and active works